charmcraft.yaml

```diff
@@ -1,26 +1,10 @@
-# Learn more about charmcraft.yaml configuration at:
-# https://juju.is/docs/sdk/charmcraft-config
-# This is a template `charmcraft.yaml` file for ops charms
-# This file is managed by bootstack-charms-spec and should not be modified
-# within individual charm repos. https://launchpad.net/bootstack-charms-spec
-
 type: charm
 
-bases:
-  - build-on:
-    - name: ubuntu
-      channel: "22.04"
-      architectures: ["amd64"]
-    run-on:
-      - name: ubuntu
-        channel: "22.04"
-        architectures:
-          - amd64
-      - name: ubuntu
-        channel: "20.04"
-        architectures:
-          - amd64
-      - name: ubuntu
-        channel: "18.04"
-        architectures:
-          - amd64
+parts:
+  charm:
+    plugin: charm
+    source: .
+
+platforms:
+  ubuntu@20.04:amd64:
+  ubuntu@22.04:amd64:
```

Comment:
Are we ok with dropping bionic support here? cc @Pjack 

If we are dropping support, we should also remove the bionic functional tests.

Reply:
Eric confirmed we can drop bionic support for all our charms, so that's all good. :+1: 

Please remove all references to the bionic func tests. :)